app/gui2/src/components/GraphEditor/widgets/WidgetPort.vue

```diff
@@ -213,4 +214,15 @@ export const widgetDefinition = defineWidget(WidgetInput.isAstOrPlaceholder, {
     right: 0px;
   }
 }
+
+.edgeAttachment {
```

Comment:
This could be a `.WidgetPort.connected:after`, then the extra div in the DOM would not be necessary.

Reply:
Implemented that myself, as well as added proper handling for dragged edges.